### PR TITLE
Canary - Filters, Labels with a single non-boolean value don't show

### PR DESCRIPTION
**Pull request by serberm**
Canary - Filters, Labels with a single non-boolean value don't show the value

### CODE REVIEW

**Comment by moshloop** [comment]
So the issue is the technology tristate toggle should include the value (springboot)

**Comment by serberm** [comment]
Could you provide more details?

File: src/components/Canary/TristateLabelStandalone/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import { TristateToggle } from "../../TristateToggle";
+import { updateParams, decodeUrlSearchParams } from "./../url";
+import { getConciseLabelState } from "./../labels";
+
+const TristateLabelStandalone = ({ label, className, labelClass, ...rest }) => {
+  const { labels: urlLabelState = {} } = decodeUrlSearchParams(
+    window.location.search
```
Comment (ciju) [inline src/components/Canary/TristateLabelStandalone/index.js:8]: Could we use react-router [useSearchParams](https://reactrouter.com/en/main/hooks/use-search-params) for url params?

File: src/components/Canary/TristateLabelStandalone/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      onChange={(v) => handleToggleChange(v)}
+      className={className}
+      labelClass={labelClass}
+      label={label}
+      {...rest}
+    />
+  );
+};
+
+export default TristateLabelStandalone;
```
Comment (ciju) [inline src/components/Canary/TristateLabelStandalone/index.js:47]: For new files, I would suggest using typescript.

**Pull request closed without merging**